src/store/actions/user.ts

```diff
@@ -0,0 +1,23 @@
+import { ActionTypes } from './action.enum';
+import { Authentication, SetRoles, SetTeam } from './action.types';
+
+
+export const reduxIsAuthentication = (isAuthentication: boolean): Authentication => {
```

Comment:
IsAuthentication means that this process is authentication. Maybe we should rename this to isAuthenticated or AuthenticationHandler etc.